infrastructure/src/babel/buildenvironment.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
```

Comment:
This looks like a duplicate of environmentbuilder.py. If it's not in use, please delete.